API/Dockerfile.dev

```diff
@@ -10,6 +10,7 @@ WORKDIR /app
 
 COPY . ./
 
+RUN dotnet dev-certs https --clean
```

Comment:
Why add this?

Reply:
To verify that we use newly generated certificates every time we build the containers.

Reply:
Makes sense, thanks!